inc/saidebugcounter.h

```diff
@@ -243,6 +243,9 @@ typedef enum _sai_in_drop_reason_t
     /** IPv6 Routing table (LPM) unicast miss */
     SAI_IN_DROP_REASON_LPM6_MISS,
 
+    /** MPLS Routing table lookup miss */
```

Comment:
Same comment as the PR related to inseg entry counter. Please add it at bottom of the list to avoid unnecessary churn in enum values.

Reply:
Moved to end of list.